frontend/src/App.js

```diff
@@ -28,59 +27,91 @@ function App() {
       const data = JSON.parse(message.data);
       const telemetry = data.telemetry
       setTelemetry(telemetry);
-      console.log(data);
+      //console.log(data);
     }
 
   },[]);
 
+  const success = () =>{
+    //Add overlay to show success message
+  }
+  const errorMessage = () =>{
+    //Add overlay to show error message
+  }
```

Comment:
rename to `displayErrorMessage`, should accept two strings as an argument.

The first argument will be `error` which is an arbitrarily defined string that is the "error type" (eg. for an invalid command, it might be "InvalidCommand" - each word capitalized with no spaces).

The second argument will be `error_message` which is another string this time as a human readable string (eg. "The provided command is invalid."). 

This will then get displayed as `<error>: <error_message>` (eg. InvalidCommand: The provided command is invalid.).

This makes it easy to search through the code for `InvalidCommand` to find which lines of code caused it.